api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,18 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
```

Comment:
I considered adding it to the stream endpoint to avoid one more round trip. Nowadays clients can directly craft a shifted offset on their side, so I though about what the performance impact would be if we forced everyone to make one more call to "materialize" a shifted offset into a non shifted offset.

I considered not publishing the extra endpoint since I also think it's redundant. But I did anyway since someone would ask for it.